packages/components/src/docs/General/6-Known Browser Issues.mdx

```diff
@@ -0,0 +1,28 @@
+import { Source, Markdown, Meta } from '@storybook/blocks';
+import playwrightVersion from 'playwright-version';
+import pkg from '../../../package.json';
+
+<Meta title="Docs/General/Known Browser Issues" />
+
+# Known Browser Issues
+
+We've identified several browser issues that may affect visual consistency while using Solid Components.
+It's important for developers to be aware of these issues when implementing our components.
+
+## Missing Border Radius for Focus Styles in Safari
+
+In Safari, focus styles may lack border radius, unlike other browsers. This
+inconsistency could affect design cohesion.
+
+## Blurry Placeholder Images in Chrome
+
+In Chrome, our placeholder images may appear blurry at times. This is not a package issue but Chrome-specific behavior.
+
+## Wobbly Spinner in Safari
+
+The 'sd-spinner' component may experience spinner animation issues in Safari, causing
+them to appear wobbly.
+
+Being aware of these browser-specific issues is essential. While we cannot
+directly resolve them, developers can make informed decisions and implement
+any necessary workarounds when using Solid Components.
```

Comment:
```suggestion
```

You can remove this sentence. Especially as some of the stuff could be fixed with ugly worksarounds, bloating our library.

Reply:
maybe we can combine the part from @Vahid1919 and add the explanation from @mariohamann as reason why we dont want to fix them (or cant fix them).

Reply:
@karlbaumhauer 
@mariohamann I have rewritten the conclusion to communicate our intention and not encourage ugly workarounds :)